ingestion/src/metadata/ingestion/source/dashboard/superset/queries.py

```diff
@@ -46,6 +47,22 @@
 	d.created_by_fk = au.id
 """
 
+FETCH_PUBLISHED_DASHBOARDS = """
+select
+	d.id, 
+	d.dashboard_title, 
+	d.position_json,
+    d.published,
+	au.email 
+from 
```

Comment:
nit - let's format the sql a bit better to have everything properly aligned and with only the necessary sapces

Reply:
Formatted the queries.

- I had 2 tab indentations setup on my editor so it was looking fine there :)